extensionsdk/nuget/Microsoft.Windows.DevHome.SDK.nuspec

```diff
@@ -7,9 +7,9 @@
     <authors>Microsoft</authors>
     <owners>Microsoft</owners>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>Dev Home SDK provides support for creating Dev Home plugins on Windows.</description>
+    <description>Dev Home SDK provides support for creating Dev Home extensions on Windows.</description>
```

Comment:
@EricJohnson327  Are we good to update this without a release imminent? Will the next one just pick it up?

Reply:
It won't get picked up until we publish a new nuget, but that should be fine.